src/Select/InputPlaceholder.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import { colors } from '../theme.js'
+
+const InputPlaceholder = ({ placeholder, className }) => {
+    if (!placeholder) {
+        return null
+    }
+
+    return (
+        <div className={className}>
+            {placeholder}
+
+            <style jsx>{`
+                div {
+                    color: ${colors.grey500};
```

Comment:
change to `grey600`

Reply:
Done in 0f89b1906b5cd2dc0b0edddb3d785ce3abd112ef